Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -64,6 +63,7 @@ public static void UnhookEvents()
                     info.Subclass.Dispose();
                 }
                 VBEEvents.Terminate();
+                _vbe = null;
```

Comment:
Do we need to dispose this first?

Reply:
No, it has been passed in from the outside. More precisely, this is the original COM wrapper from the entry point.